### DISCUSSION TITLE
plays/makdoom/typing-speed-test

### DISCUSSION THREAD

**Discussion by giscus** [reactplay/react-play#665] [Play Feedback]
# plays/makdoom/typing-speed-test

Its a typing test application in ReactJS where users can test their typing speed and accuracy.

https://reactplay.io/plays/makdoom/typing-speed-test

**Comment by atapas** [comment]
Nice play!

**Reply by atapas** [reply]
Replying on top

**Comment by makdoom** [comment]
Thanks @atapas for all your guidance and efforts..
